### ISSUE TITLE
Add set of units and magnitudes for general use

### ISSUE THREAD

**Issue by uscholdm** [semanticarts/gist#460]
Several client projects developed sets of units and magnitudes and conversions. These should be available for general use - presumably as a separate file.

**Comment by uscholdm** [comment]
DEFERRED: until there is a revision to units and magnitudes in a future major release. #697

**Comment by philblackwood** [comment]
I think we can close this issue.

**Comment by uscholdm** [comment]
I agree to close.  Things have moved on with the existence of the gist reference data
